core/src/indexer/fetch_logs.rs

```diff
@@ -197,20 +187,32 @@ async fn fetch_historic_logs_stream(
                 to_block
             );
 
-            debug!(
-                "{} - {} - Fetched {} event logs - blocks: {} - {}",
-                info_log_name,
-                IndexingEventProgressStatus::Syncing.log(),
-                logs.len(),
-                from_block,
-                to_block
-            );
-
             let logs_empty = logs.is_empty();
             // clone here over the full logs way less overhead
             let last_log = logs.last().cloned();
 
-            if tx.send(Ok(FetchLogsResult { logs, from_block, to_block })).is_err() {
+            if !logs_empty {
+                info!(
+                    "{}::{} - {} - Fetched {} logs between: {} - {}",
+                    info_log_name,
+                    network,
+                    IndexingEventProgressStatus::Syncing.log(),
+                    logs.len(),
+                    from_block,
+                    to_block
+                );
+            }
+
+            if tx.capacity() == 0 {
+                info!(
+                    "{}::{} - {} - Log channel full, indexer will backpressure (this is normal).",
```

Comment:
probs should make the log make more sense to a normal person - backpressure is not a term most people know